codespell_lib/data/dictionary.txt

```diff
@@ -9397,6 +9411,7 @@ recompence->recompense
 recomputaion->recomputation
 reconaissance->reconnaissance
 reconcilation->reconciliation
+reconfiged->reconfigured
```

Comment:
reconfiged may be an intentional misspelling., I think we should either keep and add `reconfiged` or delete this entirely.

Reply:
Removed.